Kudu.Core/Jobs/ContinuousJobRunner.cs

```diff
@@ -81,9 +81,10 @@ private void StartJob(ContinuousJob continuousJob)
 
             _continuousJobThread = new Thread(() =>
             {
-                try
+                var threadAborted = false;
+                while (!threadAborted && _started == 1 && !IsDisabled)
                 {
-                    while (_started == 1 && !IsDisabled)
+                    try
```

Comment:
this simply moves the try into the while loop making sure we will keep running continuous job as long as neither thread aborted, job stopped nor disabled.